.ci/cico_rollout_test.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+
+#This script expects this environment variables set:
+# CHE_TESTUSER_NAME, CHE_TESTUSER_PASSWORD, CHE_TESTUSER_EMAIL, RH_CHE_AUTOMATION_DEV_CLUSTER_SA_TOKEN
+
+
+# --- SETTING ENVIRONMENT VARIABLES ---
+export PROJECT=testing-rollout
+export CHE_INFRASTRUCTURE=openshift
+export CHE_MULTIUSER=true
+export CHE_OFFLINE_TO_ACCESS_TOKEN_EXCHANGE_ENDPOINT=https://auth.prod-preview.openshift.io/api/token/refresh
```

Comment:
Didn't this part change? That both API endpoints are set from a common variable?